fanet_addresses.csv

```diff
@@ -0,0 +1,3 @@
+mac_address,fanet_address
+f0:f5:bd:53:5f:20,0C0001
+00:00:00:00:00:00,0C0010
```

Comment:
Do we actually encounter this MAC address?  If so, shouldn't it be assigned a special FANET address (like 1 or 0x0CFFFF)?